utils/include/core/PropertyType.h

```diff
@@ -20,340 +20,24 @@
-
-  [[nodiscard]] PropertyValue parse(std::string_view input) const override;
-
-  [[nodiscard]] ValidationResult validate(const std::string &subject, const std::shared_ptr<minifi::state::response::Value> &input) const override {
-    return PropertyTypeImpl::_validate_internal<core::TimePeriodValue>(subject, input);
-  }
-
-  [[nodiscard]] ValidationResult validate(const std::string &subject, const std::string &input) const override {
-    auto parsed_time = utils::timeutils::StringToDuration<std::chrono::milliseconds>(input);
-    return ValidationResult{.valid = parsed_time.has_value(), .subject = subject, .input = input};
-  }
-};
-
-class DataTransferSpeedPropertyType : public PropertyTypeImpl {
- public:
-  constexpr ~DataTransferSpeedPropertyType() override {}  // NOLINT see comment at parent
-
-  [[nodiscard]] std::string_view getValidatorName() const override { return "VALID"; }
-
-  [[nodiscard]] PropertyValue parse(std::string_view input) const override;
-
-  [[nodiscard]] ValidationResult validate(const std::string &subject, const std::shared_ptr<minifi::state::response::Value> &input) const override {
-    return PropertyTypeImpl::_validate_internal<core::DataTransferSpeedValue>(subject, input);
-  }
-
-  [[nodiscard]] ValidationResult validate(const std::string &subject, const std::string &input) const override {
-    uint64_t out;
-    return ValidationResult{.valid = core::DataTransferSpeedValue::StringToInt(input, out), .subject = subject, .input = input};
-  }
-};
 
 namespace StandardPropertyTypes {
 
-inline constexpr auto INVALID_TYPE = NeverValidPropertyType{};
-inline constexpr auto INTEGER_TYPE = IntegerPropertyType{};
-inline constexpr auto UNSIGNED_INT_TYPE = UnsignedIntPropertyType{};
-inline constexpr auto LONG_TYPE = LongPropertyType{};
-inline constexpr auto UNSIGNED_LONG_TYPE = UnsignedLongPropertyType{};
-inline constexpr auto BOOLEAN_TYPE = BooleanPropertyType{};
-inline constexpr auto DATA_SIZE_TYPE = DataSizePropertyType{};
-inline constexpr auto TIME_PERIOD_TYPE = TimePeriodPropertyType{};
-inline constexpr auto NON_BLANK_TYPE = NonBlankPropertyType{};
-inline constexpr auto VALID_TYPE = AlwaysValidPropertyType{};
-inline constexpr auto PORT_TYPE = PortPropertyType{};
-inline constexpr auto LISTEN_PORT_TYPE = ListenPortValidator{};
-inline constexpr auto DATA_TRANSFER_SPEED_TYPE = DataTransferSpeedPropertyType{};
+inline constexpr auto ALWAYS_VALID_VALIDATOR = AlwaysValidValidator{};
+inline constexpr auto NON_BLANK_VALIDATOR = NonBlankValidator{};
+inline constexpr auto TIME_PERIOD_VALIDATOR = TimePeriodValidator{};
+inline constexpr auto BOOLEAN_VALIDATOR = BooleanValidator{};
+inline constexpr auto INTEGER_VALIDATOR = IntegerValidator{};
+inline constexpr auto UNSIGNED_INTEGER_VALIDATOR = UnsignedIntegerValidator{};
+inline constexpr auto DATA_SIZE_VALIDATOR = DataSizeValidator{};
+inline constexpr auto PORT_VALIDATOR = PortValidator{};
```

Comment:
Ideally, these standard validators should move to a `StandardPropertyValidators` namespace in `PropertyValidator.h`, but since that would add a lot more diffs to this already huge PR, we can do that later.

Reply:
Since thats only a drop (+80 -101) I've done it here: https://github.com/apache/nifi-minifi-cpp/pull/1926/commits/76007adaa2cb747bfe5246f1e0af159b6d96e71c

Reply:
I would also rename the namespace from `StandardPropertyTypes` to `StandardPropertyValidators`, but I still think that could go in a separate PR.

Reply:
https://github.com/apache/nifi-minifi-cpp/pull/1926/commits/14339b1288f9f7acd9bf14396645ecdda0b7f06a 🙈